src/components/TechTile.js

```diff
@@ -1,15 +1,43 @@
 import React, { isValidElement, cloneElement } from 'react';
 import PropTypes from 'prop-types';
-import { css } from '@emotion/core';
+import { ClassNames, css } from '@emotion/core';
 import { Surface, Icon, Link } from '@newrelic/gatsby-theme-newrelic';
 
 const TileIcon = ({ className, icon }) => {
   if (typeof icon === 'string') {
-    return <Icon name={icon} size="2rem" className={className} />;
+    return (
+      <Icon
+        name={icon}
+        size="2rem"
+        className={className}
+        css={css`
+          flex: 1;
+        `}
+      />
+    );
   }
 
-  if (isValidElement(icon)) {
-    return cloneElement(icon, { className });
+  if (isValidElement(icon) && icon.props?.mdxType === 'img') {
+    return (
+      <span
+        className={className}
+        css={css`
+          flex: 1;
+        `}
+      >
+        <ClassNames>
```

Comment:
Why use `ClassNames` instead of applying the `css` class directly?

Reply:
The reason is because of the `cloneElement` below. The `css` prop actually has a compile-time step to it where emotion does some work behind the scenes to turn this into a classname prop at build time. 

In this case, because this is not JSX, its difficult to use that `css` prop on `cloneElement` and have it work nicely with that build time step. This `ClassNames` utility gives you an "workaround" that allows you to attach to `className` directly. 

You can see more about this here: https://emotion.sh/docs/class-names

(though now that I re-read that, I wonder if it would work with `css`? I've had troubles in the past, but maybe thats been resolved with newer versions 🤔 )